src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1201,6 +1220,50 @@ public void joinCourseForStudent(String key, String googleId) throws JoinCourseE
         return studentsLogic.enrollStudents(enrollLines.trim(), courseId);
     
     }
+    
+    /**
+     * Sends the registration invite to unregistered students in the course.
+     * Preconditions: <br>
+     * * All parameters are non-null.
+     * 
+     * @return The list of emails sent. These can be used for
+     *         verification.
+     */
+    public List<MimeMessage> sendRegistrationInviteForCourseWithoutSendgrid(String courseId) throws InvalidParametersException {
+        Assumption.assertNotNull(ERROR_NULL_PARAMETER, courseId);
+        return studentsLogic.sendRegistrationInviteForCourseWithoutSendgrid(courseId);
+    }
+
+    /**
+     * Preconditions: <br>
+     * * All parameters are non-null.
+     */
+    public MimeMessage sendRegistrationInviteToStudentWithoutSendgrid(String courseId, String studentEmail) 
+            throws EntityDoesNotExistException, InvalidParametersException {
+        
+        Assumption.assertNotNull(ERROR_NULL_PARAMETER, courseId);
+        Assumption.assertNotNull(ERROR_NULL_PARAMETER, studentEmail);
+    
+        return studentsLogic.sendRegistrationInviteToStudentWithoutSendgrid(courseId, studentEmail);
+    }
```

Comment:
I'm not sure if we should have two sets of methods at Logic level. The caller need not know which mechanism is used to send the email. Try to subclass Emails class into two children,one each for the two mechanisms of sending emails.